.github/workflows/ci.yml

```diff
@@ -69,5 +69,5 @@ jobs:
           cd cwm-worker-helm &&\
           git add cwm-worker-deployment-minio &&\
           git commit -m "automatic update of cwm-worker-deployment-minio" &&\
-          git push origin master
+          git push origin main
```

Comment:
@iamAzeem this pushes to the helm repository which we still didn't rename to master, so it should remain with master, until we change all repositories which push there